### PR TITLE
chore: base line change, index limited 767

### PR DESCRIPTION
**Pull request by iutx**
#### What this PR does / why we need it:
base line change, index limited 767

dry run passed:
image

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | base line change, index limited 767            |
| 🇨🇳 中文    |   基线变更，索引限制767       |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: .erda/migrations/cmdb/20210528-cmdb-base.sql [hunk 1]
```diff
@@ -15,7 +15,7 @@ CREATE TABLE `cm_containers`
     `host_private_ip_addr`  varchar(255) DEFAULT NULL,
     `ip_address`            varchar(255) DEFAULT NULL,
     `image`                 varchar(255) DEFAULT NULL,
-    `cpu`                   double       DEFAULT NULL,
+    `cpu` double DEFAULT NULL,
```
Comment (sfwn) [inline .erda/migrations/cmdb/20210528-cmdb-base.sql:18]: unnecessary indent

File: .erda/migrations/cmdb/20210528-cmdb-base.sql [hunk 2]
```diff
@@ -80,13 +80,13 @@ CREATE TABLE `cm_hosts`
     `org_name`       varchar(100) DEFAULT NULL,
     `cluster`        varchar(100) DEFAULT NULL,
     `private_addr`   varchar(255) DEFAULT NULL,
-    `cpus`           double       DEFAULT NULL,
-    `cpu_usage`      double       DEFAULT NULL,
+    `cpus` double DEFAULT NULL,
+    `cpu_usage` double DEFAULT NULL,
```
Comment (sfwn) [inline .erda/migrations/cmdb/20210528-cmdb-base.sql:84]: same here

File: .erda/migrations/cmdb/20210528-cmdb-base.sql [hunk 3]
```diff
@@ -80,13 +80,13 @@ CREATE TABLE `cm_hosts`
...
     `memory`         bigint(20) DEFAULT NULL,
     `memory_usage`   bigint(20) DEFAULT NULL,
     `disk`           bigint(20) DEFAULT NULL,
     `disk_usage`     bigint(20) DEFAULT NULL,
-    `load5`          double       DEFAULT NULL,
+    `load5` double DEFAULT NULL,
```
Comment (sfwn) [inline .erda/migrations/cmdb/20210528-cmdb-base.sql:89]: same

**Review by sfwn — CHANGES_REQUESTED** [review]
I want to know what will happen if existed data exceed 191 chars, maybe truncated or migration failed?

**Comment by iutx** [comment]
> I want to know what will happen if existed data exceed 191 chars, maybe truncated or migration failed?

Migration failed and throw truncated error, e.g.
```shell
Data truncation: Data too long for column '<your_filed>' at row xxx.
```

Changed filed all limited in controlled range.

**Review by sfwn — APPROVED** [review]

**Comment by sfwn** [comment]
very nice!

/approve

**Pull request merged**